index.js

```diff
@@ -23,21 +29,26 @@ var JUnitReporter = function(baseReporterDecorator, config, emitter, logger, hel
 
   this.onRunStart = function(browsers) {
     suites = {};
-    xml = builder.create('testsuites');
 
     var suite;
     var timestamp = (new Date()).toISOString().substr(0, 19);
     browsers.forEach(function(browser) {
-      suite = suites[browser.id] = xml.ele('testsuite', {
-        name: browser.name, 'package': pkgName, timestamp: timestamp, id: 0, hostname: os.hostname()
-      });
-      suite.ele('properties').ele('property', {name: 'browser.fullName', value: browser.fullName});
+
+      pendingFileWritings++;
+
+      suite = suites[browser.id] = builder.create('testsuite');
+      suite.att('name', pkgName ? pkgName + ' / ' + browser.name : browser.name)
+           .att('timestamp', timestamp)
+           .att('hostname', os.hostname());
+      suite.ele('properties')
+           .ele('property', {name: 'browser.fullName', value: browser.fullName});
     });
   };
 
   this.onBrowserComplete = function(browser) {
     var suite = suites[browser.id];
     var result = browser.lastResult;
+    var outputFile = outputDir + 'TEST-' + browser.name.replace(/ /g, '_') + '.xml';
```

Comment:
I'm not sure removing 'TEST' do more good here since prefixing the generated report with TEST or TESTS will make the reports useful for SonarQube JavaScript Plugin.